src/scss/_header.scss

```diff
@@ -36,203 +32,148 @@ header {
       hsl(261deg 64% 59%) 100%
     );
   }
-}
 
-.header-banner {
-  position: relative;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  padding: 5em 2em;
-  width: 100%;
-  width: 80rem;
-  max-width: 100%;
-  margin: 0 auto;
-
-  @include breakpoint-up("large") {
-    padding: 5em 0;
-  }
-}
-
-//* Navigation *//
+  .header-banner {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 5em 2em;
+    width: 100%;
+    width: 80rem;
+    max-width: 100%;
+    margin: 0 auto;
 
-.dark-bg {
-  a {
-    font-weight: var(--font-weight-light);
+    @include breakpoint-up("large") {
+      padding: 5em 0;
+    }
   }
 
-  .shop-nav {
-    margin-left: 3em;
+  //* Hero *//
+  .header {
+    background-color: var(--color-primary-light);
+    padding: 6rem 0;
 
-    @media only screen and (max-width: 854px) {
-      margin-left: 0;
+    @include breakpoint-up("large") {
+      display: flex;
+      align-items: center;
     }
-  }
-}
 
-header > nav {
-  position: relative;
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: flex-end;
-  z-index: 999;
-  top: 0;
-  width: 100%;
-  min-width: 320px;
-  height: 5rem;
-  max-height: 5rem;
-
-  > a {
-    margin: 0 2.5rem;
-  }
-  > a.home-button {
-    margin-right: auto;
-  }
-  > a.home-button > img {
-    width: 10rem;
-    opacity: 1;
-  }
-  > ul {
-    height: 3rem;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    margin: 0;
-    padding: 0;
-    list-style-type: none;
-    @media only screen and (max-width: 854px) {
-      li {
-        margin: 0 0.5rem;
+    &__text {
+      @include breakpoint-up("large") {
+        flex: 2;
+        order: 1;
+        text-align: left;
+        padding: 0;
       }
     }
   }
-  > ul li {
-    margin: 0 1.2em;
-  }
 
-  > ul li a {
-    font-size: var(--font-size-lg);
-    color: var(--color-text-on-primary-lighter);
-  }
+  //* Mobile Nav Menu *//
 
-  //* Mobile nav icon *//
-
-  > input {
-    display: none;
-    margin: 0;
-    padding: 0;
-    width: 2rem;
-    height: 2rem;
-    opacity: 0;
-    cursor: pointer;
-  }
-  > label {
+  input[type="checkbox"] {
```

Comment:
Add a `cursor: pointer` style so sighted users can tell the element is clickable.